src/menu.ts

```diff
@@ -57,8 +57,10 @@ function getAppMenu(
 		},
 		{
 			label: __( 'Test Render Failure (dev only)' ),
-			click: ( menuItem, browserWindow ) => {
-				browserWindow?.webContents.send( 'test-render-failure' );
+			click: () => {
+				withMainWindow( ( window ) => {
+					window.webContents.send( 'test-render-failure' );
+				} );
```

Comment:
Out of curiosity, which change in Electron made it so we had to make this change?

Reply:
I haven't saved link to the issue and it will be difficult to find it. Long story short - they replaced BrowserWindow with BaseWindow, so in this argument we don't have access to active window anymore.

Reply:
@fredrikekelund https://github.com/electron/electron/pull/44953/files